packages/fixie/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export { FixieClient } from './client.js';
+export { IsomorphicFixieClient } from './isomorphic-client.js';
+export { FixieAgent } from './agent.js';
```

Comment:
What's the scenario where it makes sense to write code against `FixieAgent`? At the moment, it's a set of very opinionated CLI commands right now which I think we should keep separate from the rest of the client SDK.